decode/interaction.ts

```diff
@@ -17,6 +17,10 @@ export default function(tokens: Token[]): IDecodedEvent {
         case Event.TouchEnd:
         case Event.TouchMove:
             let pointerData: IPointer = { target: tokens[2] as number, x: tokens[3] as number, y: tokens[4] as number };
+            if (tokens.length > 6) {
```

Comment:
When is this sent? Add comment.

Reply:
Done, added a comment in actual code file instead of the decode. We send it when it's a click, mainly because it's an expensive operation. Later when we wish to expand this - the decoding code will remain the same.